src/App.tsx

```diff
@@ -1,7 +1,31 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends Component {
+  state = {
+    message: 'Nothing was presed yet',
```

Comment:
```suggestion
    message: 'Nothing was pressed yet',
```
tests fails because of this typo